src/data-types/objects/objects-and-arrays.js

```diff
@@ -22,10 +22,18 @@ const basket = {
 // as well as modify some of the existing code
 
 // Set this variable to the length of the baskets voucher codes array
-const numberOfVoucherCodes = null
+const numberOfVoucherCodes = 2
 
 // Set this variable to the first element in of the baskets voucher codes array
-const firstVoucherCode = null
+const firstVoucherCode = 'AA-AA-A'
+
+basket.items[0].price = 2
+
+basket.items[2] = ({
```

Comment:
there's no need for ( and ); to add an object you need to:

1) define the object with the { } syntax and
2) call push on the array:

```
baseket.items.push({...})
```